webdriver/tests/bidi/browsing_context/locate_nodes/start_nodes.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from webdriver.bidi.modules.script import ContextTarget
+
+
+@pytest.mark.parametrize("type,value", [
+    ("css", "div"),
+    ("xpath", "//div"),
+    ("innerText", "foo")
+])
+@pytest.mark.asyncio
+async def test_locate_with_context_nodes(bidi_session, inline, top_context, type, value):
+    url = inline("""<p id="parent"><div data-class="one">foo</div><div data-class="two">foo</div></p>""")
+    await bidi_session.browsing_context.navigate(
+        context=top_context["context"], url=url, wait="complete"
+    )
+
+    script_result = await bidi_session.script.evaluate(
+        expression="""document.querySelector("p")""",
+        target=ContextTarget(top_context["context"]),
+        await_promise=True,
+    )
+
+    parent_node_reference = { "sharedId": script_result["result"]["sharedId"] }
+
+    result = await bidi_session.browsing_context.locate_nodes(
+        context=top_context["context"],
+        locator={ "type": type, "value": value },
+        start_nodes=[parent_node_reference]
```

Comment:
We definitely need a test for multiple start nodes being present at different places in the markup.

Reply:
Have added a test for multiple context nodes.